### PR TITLE
Request assistance with Functions operations

### PR DESCRIPTION
**Pull request by awhitty1**
Hello,

I'm looking for feedback on my "grid_func.py" file. 
The python file is operational, however, it does contain several print statements that to an experience programmer would be unnecessary.  Welcome feedback to reduce the # of print statements.

-Alex Whitty

### CODE REVIEW

**Comment by awhitty1** [comment]
Hello,

I'm looking for feedback on my "grid_func.py" file.
The python file is operational, however, it does contain several print statements that to an experience programmer would be unnecessary. Welcome feedback to reduce the # of print statements.

-Alex Whitty

**Comment by mdgregor** [comment]
> Hello,
> 
> I'm looking for feedback on my "grid_func.py" file.
> The python file is operational, however, it does contain several print statements that to an experience programmer would be unnecessary. Welcome feedback to reduce the # of print statements.
> 
> -Alex Whitty

So you are correct, there is a lot of repetition here.  This would typically be solved with a nested `for` loop.  I don't believe these have been introduced yet so fine to omit for now.  You can see more information on them in Lesson 03 content or view some documentation on them here https://wiki.python.org/moin/ForLoop

File: students/alex_whitty/lesson2/grid_func.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+
+def plus():
+    plus = '+'
+    return plus
```
Comment (mdgregor) [inline students/alex_whitty/lesson2/grid_func.py]: This function doesn't really gain you anything as its just outputing `+`.  So on line 18 you are actually typing more by doing `plus()` than `+`

**Comment by ldconejo** [comment]
Closing this one for now, as it seems the questions Alex had have been answered by Mike.

-Luis

**Pull request closed without merging**